ssl/quic/quic_port.c

```diff
@@ -16,6 +16,10 @@
 #include "quic_channel_local.h"
 #include "quic_engine_local.h"
 #include "../ssl_local.h"
+#include <openssl/rand.h>
+
+#define EVP_GCM_IV_LEN 12
+#define EVP_GCM_TAG_LEN 16
```

Comment:
You can avoid hard coding these if you use EVP_CIPHER_get_iv_len and EVP_CIPHER_EVP_CIPHER_CTX_get_tag_len respectively below.  Should help if the cipher ever gets changed